docs/welcome/releases.md

```diff
@@ -1,23 +1,17 @@
 # Kops Releases & Versioning
 
-kops is intended to be backward compatible.  It is always recommended to use the
+Kops intends to be backward compatible.  It is always recommended using the
 latest version of kops with whatever version of Kubernetes you are using.  We suggest
 kops users run one of the [3 minor versions](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/release/versioning.md#supported-releases-and-component-skew) Kubernetes is supporting however we
-do our best to support previous releases for a period of time.
+do our best to support previous releases for some period.
 
-One exception, in regard to compatibility, kops supports the equivalent
-Kubernetes minor release number.  A minor version is the second digit in the
-release number.  kops version 1.16.0 has a minor version of 16. The numbering
-follows the semantic versioning specification, MAJOR.MINOR.PATCH.
+Kops does not, however, support Kubernetes releases with a greater minor release number
+than that of kops. A minor version is the second digit in the
+release number: kops version 1.16.0 has a minor version of 16. The numbering
+follows the semantic versioning specification: MAJOR.MINOR.PATCH.
```

Comment:
Worth linking out to the semver spec here?

Reply:
The semver spec would be highly parenthetical. The key point here is that it is the second part that counts.

Though now I notice it's not the second _digit_ and, pedantically, the major version number also matters.